### DISCUSSION TITLE
Welcome to Discussions!

### DISCUSSION THREAD

**Discussion by Ralim** [Ralim/IronOS#729] [General]
## 👋 Welcome!
Going to try this so that we can have less noise in the issues section.
This is oriented more towards forum style use :)
Still keep bugs in the Issues tab :)

**Comment by discip** [comment]
@Ralim
I think in general this is a very good thing! Especially the ability to vote! 😃👍
But I think it would be nice, if it was introduced to the people like myself, who use the path they already know almost exclusively. 😁
As you stated above:
> less noise in the issues section.

To be more precise, may be it would help to make this *'new way'* more public by announcing it in the **`issue`** section.

**Reply by mark1352** [reply]
Mhp30 will iron os run on this

**Reply by Ralim** [reply]
MHP30 is supported by IronOS and there are already MHP30 builds available in releases.
All flashing instructions are the same as existing Miniware products.
